### DISCUSSION TITLE
Add package.json > biome config support

### DISCUSSION THREAD

**Discussion by chneau** [biomejs/biome#2061] [Ideas]
Prettier configuration can be set in package.json instead of creating "yet" another file in the root of the project.
I believe in VSCode, when you have the prettier extension, you get the autocompletion on package.json>prettier.

Would it be nice to have the same UX for biome?

(Couldn't transfer from https://github.com/biomejs/biome-vscode/discussions/169)

**Comment by arendjr** [comment]
Hah, I think I transferred it at the same time as you created this one.

**Comment by arendjr** [comment]
I’ll close as duplicate.
